### ISSUE TITLE
names can't set tmux's session in the lenovo

### ISSUE THREAD

**Issue by schneiderfelipe** [schneiderfelipe/dotfiles#3]
![Screenshot from 2022-04-25 17-48-48](https://user-images.githubusercontent.com/37125/165173088-3e863e36-c178-4793-a90c-3737488c8b97.png)

**Comment by schneiderfelipe** [comment]
On apollo14 (the computer where things work fine), I get the following:

```console
$ whereis names
names: /usr/local/bin/names
$ names --version
names 0.12.0
$ names
alcoholic-drop
```

**Comment by schneiderfelipe** [comment]
Solved by 5a1835f0d9bf884ca8c107477b8800f29cff217d
